apps/store/src/components/Perils/Perils.stories.tsx

```diff
@@ -18,47 +20,58 @@ const Template: ComponentStory<typeof Perils> = (props) => <Perils {...props} />
-      covered: ['Lorem ipsum dolor sit amet', 'Sed fermentum tempus'],
-      notCovered: [
-        'Morbi at egestas tortor',
-        'Morbi vitae elit sapien',
-        'Duis sed viverra nibh',
-        'Quisque venenatis lacus dolor',
-      ],
-    },
-    {
-      id: 'storms',
-      icon: <ShieldIcon size="22px" />,
-      name: 'Storms',
-      description:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent at dictum urna. Pellentesque gravida, sapien ut maximus cursus, dui ligula sodales nisl, sed placerat felis metus quis dolor.',
-      covered: [
-        'Lorem ipsum dolor sit amet',
-        'Sed fermentum tempus',
-        'Quisque venenatis lacus dolor',
-      ],
-      notCovered: ['Morbi vitae elit sapien'],
-    },
-  ],
+  product: {
+    name: ProductNames.SE_CAR,
+    displayName: 'Car insurance',
+    market: MarketLabel.SE,
+    insurances: [
+      {
+        name: InsuranceNames.SE_CAR,
+        displayName: 'Car insurance',
+        perils: [
+          {
+            id: 'waterLeaks',
+            icon: <ShieldIcon size="22px" />,
+            name: 'Water leaks',
+            description:
+              'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent at dictum urna. Pellentesque gravida, sapien ut maximus cursus, dui ligula sodales nisl, sed placerat felis metus quis dolor.',
+            covered: [
+              'Lorem ipsum dolor sit amet',
+              'Sed fermentum tempus',
+              'Morbi at egestas tortor',
+              'Quisque venenatis lacus dolor',
+            ],
+            notCovered: ['Morbi vitae elit sapien', 'Duis sed viverra nibh'],
+          },
+          {
+            id: 'fire',
+            icon: <ShieldIcon size="22px" />,
+            name: 'Fire',
+            description:
+              'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent at dictum urna. Pellentesque gravida, sapien ut maximus cursus, dui ligula sodales nisl, sed placerat felis metus quis dolor.',
+            covered: ['Lorem ipsum dolor sit amet', 'Sed fermentum tempus'],
+            notCovered: [
+              'Morbi at egestas tortor',
+              'Morbi vitae elit sapien',
+              'Duis sed viverra nibh',
+              'Quisque venenatis lacus dolor',
+            ],
+          },
+          {
+            id: 'storms',
+            icon: <ShieldIcon size="22px" />,
+            name: 'Storms',
+            description:
+              'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent at dictum urna. Pellentesque gravida, sapien ut maximus cursus, dui ligula sodales nisl, sed placerat felis metus quis dolor.',
+            covered: [
+              'Lorem ipsum dolor sit amet',
+              'Sed fermentum tempus',
+              'Quisque venenatis lacus dolor',
+            ],
+            notCovered: ['Morbi vitae elit sapien'],
+          },
+        ],
+      },
+    ],
+  },
```

Comment:
Could we use `PRODUCT` from mockservice here as well? :)
```suggestion
  product: PRODUCTS[1],
}
```